ui/components/ui/tabs/tabs.component.js

```diff
@@ -47,11 +47,11 @@ export default class Tabs extends Component {
     const numberOfTabs = React.Children.count(this._getValidChildren());
 
     return React.Children.map(this._getValidChildren(), (child, index) => {
-      const tabName = child?.props.name;
+      const tabKey = child?.props.tabKey;
       return (
         child &&
         React.cloneElement(child, {
-          onClick: (idx) => this.handleTabClick(idx, tabName),
+          onClick: (idx) => this.handleTabClick(idx, tabKey),
           tabIndex: index,
           isActive: numberOfTabs > 1 && index === this.state.activeTabIndex,
```

Comment:
It's possible the e2e test failure is because `t` is undefined in `tab.component.js` when a tab is rendered via this method

Reply:
Good catch !!